buildSrc/src/main/groovy/org/elasticsearch/gradle/vagrant/VagrantTestPlugin.groovy

```diff
@@ -142,10 +144,12 @@ class VagrantTestPlugin implements Plugin<Project> {
     }
 
     private static void createCleanTask(Project project) {
-        project.tasks.create('clean', Delete.class) {
-            description 'Clean the project build directory'
-            group 'Build'
-            delete project.buildDir
+        if (project.tasks.findByName('clean') == null) {
```

Comment:
The java plugin also adds a clean task, and it complains when we try to add another one. My thinking was that I didn't want to bake in the expectation that the project vagrant test plugin is applied to is also a java project. For example `:x-pack:qa:vagrant` has the vagrant test plugin but not the java plugin

Reply:
Hmmm. Maybe maybe it'd be simpler to create a noop java test in there in this PR instead of doing this? I imagine we'll want to make java tests in there eventually anyway.

Reply:
I believe we're going to put all the java packaging tests (including x-pack) into `:qa:vagrant`, and once all the bats tests are ported we can remove `:x-pack:qa:vagrant`